sys/include/net/gnrc/tcp.h

```diff
@@ -34,6 +34,39 @@
 extern "C" {
 #endif
 
+
+/**
+ * @brief Address information for a single TCP connection endpoint.
+ * @note Must be compatible to sock_tcp_ep_t.
+ */
+typedef struct {
+    int family; /**< IP Address family. GNRC supports currently only IPv6. */
+
+    union {
+#ifdef MODULE_GNRC_IPV6
+        uint8_t ipv6[sizeof(ipv6_addr_t)]; /**< IPv6 Address storage */
+#endif
+    } addr;
+
+    uint16_t netif; /**< Network interface ID */
+    uint16_t port;  /**< Port number (in host byte order) */
+} gnrc_tcp_ep_t;
+
+
+/**
+ * @brief Initialize TCP connection endpoint.
+ *
+ * @param[in,out] ep               Endpoint to initialize.
+ * @param[in]     address_family   Address family of @p address.
+ * @param[in]     address          Address information for endpoint.
+ * @param[in]     port             Port number for endpoint.
+ *
+ * @returns   0 on success.
+ *            -EAFNOSUPPORT if @p address_family is not supported.
```

Comment:
```suggestion
 * @returns   -EAFNOSUPPORT if @p address_family is not supported.
```

to make it tabular in rendered doc.

Reply:
I assume this is something that should be changed throughout the entire API right?

Reply:
Yes, please :-).